src/Indigo/Contracts/HomebaseLite/Types.hs

```diff
@@ -0,0 +1,67 @@
+-- SPDX-FileCopyrightText: 2022 Tezos Commons
+-- SPDX-License-Identifier: LicenseRef-MIT-TC
+
+{-# OPTIONS -Wno-orphans #-}
+
+module Indigo.Contracts.HomebaseLite.Types
+  ( Parameter(..)
+  , Storage(..)
+  , Configuration(..)
+  , URI(..)
+  ) where
+
+import Indigo
+
+data Parameter
+  = Set_admin KeyHash
+  | Accept_admin ()
+  | Add_maintainers [KeyHash]
+  | Remove_maintainers [KeyHash]
```

Comment:
:thinking: I remember now that we discussed about making these Sets and that I provided some explanation for why lists are more often used.

However I'm not particularly against set in this case and I don't think that the `list` in the spec should be taken too literally (or it could be updated).

What do you think?

Reply:
At the end of the day, it doesn't really matter for the implementation, so... I don't know, it doesn't seem like there's much point to using a set.

Reply:
Yeah, the only advantage is the one you noted of not permitting duplicates in the parameter with sets.

However admittedly this doesn't change the logic (even if there are duplicates the result will be the same) and it will only add a cost for those that are not careful enough.

Either option sounds ok to me as well :shrug: